packages/ckeditor5-bookmark/src/bookmarkediting.ts

```diff
@@ -111,4 +123,22 @@ export default class BookmarkEditing extends Plugin {
 			return domElement;
 		} );
 	}
+
+	private upcastMatcher( element: ViewElement ) {
```

Comment:
This does not need to be exposed as a plugin public API. As it's using only parameters without side-effects we could use a plain helper function here (instead of method). That helper could be even not exported.